Dockerfile

```diff
@@ -1,13 +1,44 @@
-FROM ministryofjustice/ruby:2.5.1-webapp-onbuild
+FROM ministryofjustice/ruby:2.5.1
 
-# Ensure the pdftk package is installed as a prereq for ruby PDF generation
-ENV DEBIAN_FRONTEND noninteractive
+# add official nodejs repo
+RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
+        echo 'deb https://deb.nodesource.com/node jessie main' > /etc/apt/sources.list.d/nodesource.list
 
-EXPOSE 8080
+# install runit and nodejs
+RUN apt-get update && apt-get install -y runit nodejs && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && rm -fr *Release* *Sources* *Packages* && \
+    truncate -s 0 /var/log/*log
+
+# SSH proxy settings
+ENV SSH_AUTH_SOCK /tmp/ssh-auth
+ENV SSH_AUTH_PROXY_PORT 1234
+
+RUN mkdir -p /usr/src/app
+RUN bundle config --global without test:development
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+COPY ./vendor /usr/src/app/vendor
+
+ONBUILD COPY Gemfile /usr/src/app/
+ONBUILD COPY Gemfile.lock /usr/src/app/
 
+# Hack to install private gems
+ONBUILD RUN socat UNIX-LISTEN:$SSH_AUTH_SOCK,fork TCP4:$(ip route|awk '/default/ {print $3}'):$SSH_AUTH_PROXY_PORT & bundle install
+
+ONBUILD COPY . /usr/src/app
```

Comment:
You have COPY . /usr/src/app already - I dont think these ONBUILD instructions will do anything - check with devops but im sure they are only used when another Docker file uses yours which obviously wont happen